plugins/DuskVerb/tools/tuner/full_check.py

```
@@ -543,6 +606,132 @@ def energy_arrival(p, drop_db=40.0, window_ms=2000.0):
+    """List of DISCRETE early reflections FOLLOWING the transient, as (t_ms,
+    prominence_dB) onset-relative, within the first win_ms.
+
+    Where early_tap reports the single DOMINANT arrival (argmax), this reports
+    the whole PATTERN: peak-pick the 3 ms-smoothed Hilbert envelope (dB) with a
+    prominence floor (audibly discrete), a minimum inter-peak spacing, and an
+    absolute height floor (onset_peak - floor_db) so dither/quantization wiggle
+    in quiet windows can't fabricate phantom taps. The search starts gap_ms past
+    the onset so the transient itself is NOT counted — only the reflections that
+    follow it. A struck transient + discrete slaps yields several entries; a
+    smooth diffuse swell yields ~zero. (None if the window is too short.)"""
+    x, sr = sf.read(p); m = x.mean(axis=1) if x.ndim > 1 else x
```

Ruff (0.15.18)

[error] Multiple statements on one line (semicolon)

(E702)